pandas/tests/extension/test_arrow.py

```diff
@@ -1337,6 +1337,28 @@ def test_quantile(data, interpolation, quantile, request):
     data = data.take([0, 0, 0])
     ser = pd.Series(data)
     result = ser.quantile(q=quantile, interpolation=interpolation)
+
+    if pa.types.is_timestamp(pa_dtype) and interpolation not in ["lower", "higher"]:
+        # rounding error will make the check below fail
```

Comment:
For the rounding errors for `interpolation not in ["lower", "higher"]`, is it due to the int casting or just the quantile computation?

Reply:
not sure where it comes in.  just checked that the results match what we get with DTA